allennlp/service/static/index.html

```diff
@@ -364,7 +364,7 @@
     const { outputState, runModel, inputs } = this.props;
 
     return (
-    <button id="input--mc-submit" type="button" disabled={outputState === "working"} className="btn btn--icon-disclosure" onClick={ () => { runModel(event, inputs) }}>Run
+    <button id="input--mc-submit" type="button" disabled={outputState === "working"} className="btn btn--icon-disclosure" onClick={ (event) => { runModel(event, inputs) }}>Run
```

Comment:
This is the only change that necessary.

Reply:
Thanks @bbstilson -- you're right.  I got a bit confused using the FireFox developer tools.  That we have everything in on mega-file with babel doesn't help the errors and line numbers any...

Reply:
For sure. To be clear, this isn't a firefox issue. It's just a reference error 🙂

Reply:
My understanding is that Chrome (and IE and Safari) have a global `event` variable that they update.

Reply:
Ah, I see. Browsers are the worst.

Reply:
Yea, you don't want to rely upon the global `event`.  I've never hard of that -- nor would I suggest using it.  Globals are bad :)